Dockerfile

```diff
@@ -19,6 +19,11 @@ RUN yum -y install epel-release \
     tango-db \
     tango-accesscontrol \
     tango-test \
+    python-pytango \
+    gcc \
```

Comment:
Oh, okay, I thought that `pymodbus` doesn't require `gcc` (which is a C compiler), but it turns out that `twisted`, the networking library, needs it. Leave it like that.